### ISSUE TITLE
study_attribute entity type throws 500 error

### ISSUE THREAD

**Issue by cboettig** [movebank/movebank-api-doc#2]
Looks like any request I make against the `study_attribute` entity type endpoint throws an internal server error, (500).  for example, this query:

```
https://www.movebank.org/movebank/service/direct-read?entity-type=study_attribute&study_id=2238275
```

throws a 500 error.  I don't have trouble querying against other endpoints like `study` and `taxon`.  Any idea why this is happening? Apologies if this is not the right place to report the error.

**Comment by xhdong-umd** [comment]
You need to add `sensor_type_id` like this:

`https://www.movebank.org/movebank/service/direct-read?entity_type=study_attribute&study_id={{buffalo}}&sensor_type_id=653`

You can find the `sensor_type_id` through other calls first.

It often require some hidden requirements not fully documented, though I think I have documented the most of them in the document.

You can have a look at [my repo here](https://github.com/xhdong-umd/movebank-api-doc), which have a collection of [postman](https://www.getpostman.com/), included many tried and true examples. 

<img width="286" alt="screen shot 2017-08-25 at 2 22 11 pm" src="https://user-images.githubusercontent.com/25039897/29727213-f17fb46c-89a0-11e7-8d00-40db301ff78e.png">

Note I have to remove some login info from the collection, see more details in the readme.md of my repo.

**Comment by xhdong-umd** [comment]
Yes, I did document this usage in [another file](https://github.com/xhdong-umd/movebank-api-doc/blob/master/document/all_attributes.md#entity-typestudy_attribute)

> Need study_id and sensor_type_id in call. Get sensor_type_id from sensor call.
> 
> https://www.movebank.org/movebank/service/direct-read?entity_type=study_attribute&study_id=1764627&sensor_type_id=653
